client_vpn/input.tf

```
@@ -120,3 +107,21 @@ variable "vpc_id" {
   description = "(Required) ID of the VPC to associate with the VPN endpoint."
   type        = string
 }
+variable "authentication_option" {
+  description = "(Optional, default 'federated-authentication') The authentication option to use for the VPN endpoint.  Valid values are 'federated-authentication' or 'certificate-authentication'."
+  type        = string
+  default     = "federated-authentication"
+}
+variable "client_vpn_saml_metadata_document" {
+  description = "(Required) The base64 encoded SAML metadata document for the Client VPN endpoint"
```

Minor tweak just to the description now that there's a default value:
```suggestion
  description = "(Optional, default empty) The base64 encoded SAML metadata document for the Client VPN endpoint"
```